ts/features/wallet/onboarding/bancomat/screens/add-pans/AddBancomatScreen.tsx

```diff
@@ -0,0 +1,131 @@
+import * as pot from "italia-ts-commons/lib/pot";
+import * as React from "react";
+import { View } from "native-base";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { SafeAreaView } from "react-native";
+import { fromNullable } from "fp-ts/lib/Option";
+import { GlobalState } from "../../../../../../store/reducers/types";
+import BaseScreenComponent from "../../../../../../components/screens/BaseScreenComponent";
+import { abis, pans } from "../../mock/mockData";
+import { H1 } from "../../../../../../components/core/typography/H1";
+import customVariables from "../../../../../../theme/variables";
+import FooterWithButtons from "../../../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  confirmButtonProps
+} from "../../../../../bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import { navigateBack } from "../../../../../../store/actions/navigation";
+import I18n from "../../../../../../i18n";
+import { Card } from "../../../../../../../definitions/pagopa/bancomat/Card";
+import { profileSelector } from "../../../../../../store/reducers/profile";
+import { H4 } from "../../../../../../components/core/typography/H4";
+// import { abiListSelector } from "../../store/reducers/abi";
+import PanCardComponent from "./PanCardComponent";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+/**
+ * This screen is displayed when Bancomat are found and ready to be added in wallet
+ * @constructor
+ */
+const AddBancomatScreen: React.FunctionComponent<Props> = (props: Props) => {
+  const [currentIndex, setCurrentIndex] = React.useState(0);
+  const [abiLogo, setAbiLogo] = React.useState<string | undefined>();
+
+  const handleOnContinue = () => {
+    if (currentIndex === props.pans.length - 1) {
```

Comment:
```suggestion
    if (currentIndex < props.pans.length) {
      setCurrentIndex(currentIndex + 1);
    } 
    props.onContinue();
```

Reply:
@Undermaken This is just a placeholder function, this will be completely reworked when adding the store/business logic

Reply:
Ok. In these cases could be useful to have a `TODO` comment